avm/res/key-vault/vault/access-policy/main.bicep

```diff
@@ -6,13 +6,13 @@ metadata owner = 'Azure/module-maintainers'
 param keyVaultName string
```

Comment:
Please update the module using the `avm/utilities/Set-AVMModule.ps1` utility. It will update the ReadMes & JSONs for you :)

Reply:
Done!